### PR TITLE
First round of cleanup to start release 1 sprint.

### PR DESCRIPTION
**Pull request by aiuto**
- Add deps.bzl (properly depend on python)
- Add license rule for entire package.
- Update checker_demo to account for bazel fixes to UTF-8 support
- Add some missing copyright notices

There is certainly more to do. I am breaking work into either logical
or 1 day chunks.

- [X] Tests pass
- [X ] Appropriate changes to README are included in PR

### CODE REVIEW

File: rules/BUILD [hunk 1]
```diff
@@ -13,6 +13,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
+
 # TODO(aiuto): Add dependency on Skylib for bzl_library so that we can later
 # generate docs with stardoc.
+
+load("@rules_license//rules:license.bzl", "license")
```
Comment (danielmachlab) [inline rules/BUILD:20]: The `license` rule is loaded but is unused, this line can be removed.
```suggestion
```

File: licenses/spdx/BUILD [hunk 2]
```diff
@@ -38,9 +52,15 @@
 # If you maintain your own  third_party directory of all your dependences,
 # so that your WORKSPACE files are empty, you might favor the second.
 
-
+load("@rules_license//rules:license.bzl", "license")
```
Comment (danielmachlab) [inline licenses/spdx/BUILD:55]: The `license` rule is loaded but is unused, this line can be removed.
```suggestion
```

**Review by danielmachlab — CHANGES_REQUESTED** [review]
Hi Tony, I've enjoyed getting to play around with rules_license over the last couple months. Excited to get to jump in and help by review PRs.

File: tools/BUILD [hunk 3]
```diff
@@ -1,4 +1,25 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# License declaration and compliance checking tools.
+
+load("@rules_license//rules:license.bzl", "license")
+
+package(
+    default_applicable_licenses = [":license"],
```
[the remark this answers is not in the document]
Comment (aiuto) [reply tools/BUILD:20]: Whoops. That is a typo.  It should be //:license. Fixed.

File: rules/BUILD (hunk 1 above)
Comment (aiuto) [reply rules/BUILD]: Yes. And removed in the other places as well.

File: licenses/spdx/BUILD (hunk 2 above)
Comment (aiuto) [reply licenses/spdx/BUILD]: done

**Comment by aiuto** [comment]
Thanks for catching the typos. I am looking forward to doing a quick sprint to usability in the Bazel source tree.

**Pull request merged**